library/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
     compile('com.android.support.test.espresso:espresso-contrib:2.0') {
         exclude module: 'support-v4'
     }
+    compile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
+    compile 'com.android.support:appcompat-v7:25.1.0'
```

Comment:
Look at the code of that method. It's one line with a nullcheck. Maybe we can copy that code and avoid the ContextCompat and remove the support library?